connectors/sources/github.py

```diff
@@ -2075,6 +2076,24 @@ async def _fetch_access_control(self, repo_name):
                         access_control.append(_prefix_email(email=user_email))
         return access_control
 
+    def _is_repo_processed(self, repo_name):
+        if self._checkpoint is None:
+            self._logger.debug(f"{repo_name} was not processed by this job before.")
+            return False
+
+        self._logger.info(
+            f"Skipping repository {repo_name} - has already been processed."
+        )
+        return self._checkpoint.get(repo_name)
```

Comment:
How do we know it's been processed, before checking  the checkpoint?

Reply:
It's a mistake haha, good catch